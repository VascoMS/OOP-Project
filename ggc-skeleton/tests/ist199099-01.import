PARTNER|A1|Luís Câmara|Lisboa, Portugal
BATCH_S|H|A1|1|10000
BATCH_S|O|A1|1|10000
BATCH_M|ÁGUA|A1|3|1|0|H:2#O:1
BATCH_S|VIDRO|A1|1|10000
BATCH_M|GARRAFA|A1|303|1|0|ÁGUA:100#VIDRO:10
