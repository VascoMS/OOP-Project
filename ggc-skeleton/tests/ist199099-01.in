7
3
A1
15
GARRAFA
20
0
5
2
1
0
6
2
6
A1
0
7
1
0
0
0
