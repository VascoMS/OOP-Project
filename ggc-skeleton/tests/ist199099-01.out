Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Produtos
1 - Visualizar todos os produtos
2 - Visualizar todos os lotes
3 - Visualizar lotes fornecidos por parceiro
4 - Visualizar lotes de um dado produto
0 - Sair
Escolha uma opção: H|A1|1|6202
O|A1|1|8101
VIDRO|A1|1|9810
Gestão de Produtos
1 - Visualizar todos os produtos
2 - Visualizar todos os lotes
3 - Visualizar lotes fornecidos por parceiro
4 - Visualizar lotes de um dado produto
0 - Sair
Escolha uma opção: GARRAFA|310|0|ÁGUA:100#VIDRO:10
H|1|6202
O|1|8101
VIDRO|1|9810
ÁGUA|3|0|H:2#O:1

Gestão de Produtos
1 - Visualizar todos os produtos
2 - Visualizar todos os lotes
3 - Visualizar lotes fornecidos por parceiro
4 - Visualizar lotes de um dado produto
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: A1|Luís Câmara|Lisboa, Portugal|NORMAL|0|0|6193|0
Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Identificador do parceiro: VENDA|0|A1|GARRAFA|20|6193|5574|15

Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador da transacção: VENDA|0|A1|GARRAFA|20|6193|5574|15

Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: 